src/qrcore.h

```diff
@@ -97,6 +112,8 @@ public slots:
     QString default_arch;
     QString default_cpu;
     int default_bits;
+
+    RCore *core_;
```

Comment:
I find the amout of `core`s confusing and `core_` ugly. Why not call it `rcore` or `m_rcore` or something like that?